guides/manual/1-overview.md

```diff
@@ -1,71 +1,151 @@
-
-Usage across various rendering frameworks will be similar. In fact, this is an
-explicit goal: WarpDrive enables developers to quickly port and re-use data
-patterns.
-
-We see this as one of the keys to scalability. Providing a stable framework
-for how data is requested, cached, mutated, and mocked allows developers to
-focus more time on the product requirements that matter.
-
-A single WarpDrive configuration can power multiple web-apps using differing
-frameworks all sharing a single domain: bridging the gap between MPA and SPA.
-
-<br>
-
-## A Very Brief History
-
-WarpDrive began in ~2006 as a suite of ORM-like data utilities in SproutCore that
-later evolved into EmberData. Beginning in ~2017 the team plotted a course towards
-a significant evolution of the library based on the lessons learned in the first
-eleven years.
-
-In ~2023 we lifted the veil on that work, shifting from a resource-centric library
-to a request-centric library and began accelerating towards our goal of becoming
-a universal library prepared for the demands of web applications over the coming
-decades.
-
-<br>
-
-## Why WarpDrive?
+## Overview
 
 WarpDrive is the data framework for building ambitious applications.
 
-What do we mean by ambitious? WarpDrive is ideal for applications looking to
-be best-in-class: whether that's a small todo app, e-commerce, a
-social app, or an enterprise b2b software solution.
+What do we mean by ambitious? WarpDrive is ideal for applications looking to be best-in-class: whether that's a small todo app, e-commerce, a social app, or an enterprise b2b software solution.
+
+The capabilities WarpDrive provides are designed to seamlessly handle and simplify the hardest parts of state management when building an app, helping you focus on creating the features and user experiences that drive value.
+
+Its innovative approach to [fine grained reactivity](https://dev.to/ryansolid/a-hands-on-introduction-to-fine-grained-reactivity-3ndf) enables rapidly developing robust, performant web applications using any [Signals](https://github.com/tc39/proposal-signals#readme) compatible framework such as [Ember](https://guides.emberjs.com/release/in-depth-topics/autotracking-in-depth/), [Svelte](https://svelte.dev/docs/svelte/what-are-runes), [Angular](https://angular.dev/guide/signals), [Vue.js](https://vuejs.org/guide/extras/reactivity-in-depth.html), [SolidJS](https://www.solidjs.com/tutorial/introduction_signals),
+[Preact](https://preactjs.com/guide/v10/signals/) or [Lit](https://lit.dev/docs/data/signals/).
+
+```hbs
+  Hello {{@user.name}}!
+```
+
+By building around the same interface as the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), WarpDrive makes powerful request management features like caching, deduping and data normalization feel simple to use. We pair this with [reactive control flow](./concepts/reactive-control-flow.md) to give apps the ability to declaratively derive states with safety.
+
+```glimmer-ts
```

Comment:
Loving the template-only example. Really helps to illustrate how little you have to do to integrate.